src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -83,7 +83,7 @@ import { PTSD, PTSD_INCIDENT_ITERATION } from '../constants';
 
 import fullSchema from './schema';
 
-const formConfig = {
+const createFormConfig = {
```

Comment:
I remember making the comment about changing the name of a function, but this isn't a function anymore? Either way, it could probably go back to just being called `formConfig`, because it's not creating anything.

Reply:
Ughh...squash and merge strikes again. Yeah this needs to just be `formConfig`